package.json

```diff
@@ -79,14 +79,15 @@
     "postcss-import": "^12.0.1",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",
+    "postcss-safe-parser": "^4.0.2",
     "prettier": "^2.4.1",
     "prop-types": "^15.7.2",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-helmet-async": "^1.0.4",
     "react-router-dom": "^5.2.0",
     "style-loader": "^0.23.1",
-    "stylelint": "^11.1.1",
+    "stylelint": "^14.2.0",
```

Comment:
What is the reason that we needed to upgrade?

Reply:
`stylelint` didn't add the `alpha-value-notation` rule until [13.10.0](https://stylelint.io/changelog/#13100)